app/assets/stylesheets/responsive/custom.scss

```diff
@@ -1917,4 +1917,10 @@ $color_asktheeu_dark_green: #C4DDB9;
     display: none;
     visibility: hidden;
   }
-}
\ No newline at end of file
+}
+
+.act-link-icon {background-size: 144px 16px;}
+
+.act-link-icon--donation {
+ background-position: -128px 0;
```

Comment:
I think the rest of this file uses a 2 space indent, rather than the 1 space at the start of this line.